src/zino/statemodels.py

```diff
@@ -179,9 +179,9 @@ class BGPEvent(Event):
 
 class BFDEvent(Event):
     type: Literal["bfd"] = "bfd"
-    bfdix: Optional[int] = None
-    bfddiscr: Optional[int] = None
-    bfdaddr: Optional[IPAddress] = None
+    ifindex: Optional[int] = None
+    session_index: Optional[int] = None
+    session_state: Optional[BFDSessState] = None
```

Comment:
For a re-implementation it makes sense to give these more reasonable names, but be aware that these names are taken from the actual API protocol.  If we change these names internally, we must also add aliases to these fields so that they are still exposed under their old names by the server protocol.

Reply:
Probably not worth the hassle then, I'll change the names back